create-repo.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import re
+import subprocess
+import sys
+import textwrap
+
+import requests
+
+sparql_endpoint = 'https://query.wikidata.org/sparql'
+iso_3166_1_code = sys.argv[1].lower()
+
+if os.path.isdir(iso_3166_1_code):
+    raise AssertionError(f'Country repository already exists: {iso_3166_1_code}')
+
+query = f'''\
+SELECT * WHERE {{
+  ?country wdt:P297 '{iso_3166_1_code.upper()}' ;
+           wdt:P37/wdt:P424 ?language ;
+           rdfs:label ?label .
+  FILTER(LANG(?label) = 'en')
+}}
+'''
+
+print(query)
+
+response = requests .post(sparql_endpoint, query,
+                          headers = {'Content-Type': 'application/sparql-query',
+                                     'Accept': 'application/sparql-results+json'})
+response.raise_for_status()
+
+results = response.json()
+bindings = results['results']['bindings']
+
+languages = {binding['language']['value'] for binding in bindings}
+languages = sorted(languages)
+if 'en' not in languages:
+    languages.append('en')
+
+country_wikidata_id = bindings[0]['country']['value'].split('/')[-1]
```

Comment:
It might be worth checking that there's exactly one `?country` found by the query (and erroring otherwise) before picking the country from the first result row. (I haven't checked, but I bet that at some point there'll be multiple items given the same `P279` value...)

Reply:
I've added explicit checks for 0 and >1 countries, with appropriate messages.